src/config.rs

```diff
@@ -77,7 +77,7 @@ fn memory_mb_for_cf(is_raft_db: bool, cf: &str) -> usize {
 #[serde(default)]
 #[serde(rename_all = "kebab-case")]
 pub struct TitanCfConfig {
-    pub min_blob_size: u64,
+    pub min_blob_size: ReadableSize,
```

Comment:
Does it compactable with pervious config?

Reply:
Yes